environment.yml

```
@@ -10,12 +10,13 @@ dependencies:
   - openms::openms=2.7.0pre
   - openms::openms-thirdparty=2.7.0pre # 08.02.2021
   - bioconda::bioconductor-msstats=3.20.1 # will include R
-  - bioconda::sdrf-pipelines=0.0.13 # for SDRF conversion
+  - bioconda::sdrf-pipelines=0.0.14 # for SDRF conversion
   - conda-forge::r-ptxqc=1.0.5 # for QC reports
   - conda-forge::xorg-libxt=1.2.0 # until this R fix is merged: https://github.com/conda-forge/r-base-feedstock/pull/128
   - conda-forge::fonts-conda-ecosystem=1 # for the fonts in QC reports
   - conda-forge::python=3.8.5
   - conda-forge::markdown=3.2.2
   - conda-forge::pymdown-extensions=8.0.1
   - conda-forge::pygments=2.7.1
+  - conda-forge::mono=5.14.0.177
```

```suggestion
  - conda-forge::mono=5.14.0.177
  - conda-forge::pandoc=2.11.4
```

Thanks conda for another incompatibility error. Wherever this comes from?!

---

Just released yesterday, already breaks things.